modules/storage-persistent-storage-pv.adoc

```diff
@@ -179,8 +179,10 @@ endif::[]
 
 3. Use a recreate deployment strategy for pods that rely on AWS EBS.
 
+4. Only raw block volumes support ReadWriteMany access mode for Fibre Channel and iSCSI. For more information, see "Block volume support".
+
 ifndef::openshift-dedicated,openshift-rosa[]
-4. If the underlying vSphere environment supports the vSAN file service, then the vSphere Container Storage Interface (CSI) Driver Operator installed by
+5. If the underlying vSphere environment supports the vSAN file service, then the vSphere Container Storage Interface (CSI) Driver Operator installed by
 {product-title} supports provisioning of ReadWriteMany (RWX) volumes. If you do not have vSAN file service configured, and you request RWX, the volume fails to get created and an error is logged. For more information, see "Using Container Storage Interface" -> "VMware vSphere CSI Driver Operator".
```

Comment:
```suggestion
{product-title} supports provisioning of RWX volumes. If you do not have vSAN file service configured, and you request RWX, the volume fails to get created and an error is logged. For more information, see "Using Container Storage Interface" -> "VMware vSphere CSI Driver Operator".
```

Reply:
In `4.` we mentioned "ReadWriteMany (RWX)" so here only "RWX" can be kept.

Reply:
This is outside of the scope of the change suggested in this jira. Also I think it is good to keep the expansion and short form (ReadWriteMany (RWX)) both as it helps user.